content/en/kb/install-lotus-wallet/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "How to install lotus-wallet"
+description: "This is a guide for installing lotus-wallet."
+date: 2022-08-15T12:00:35+01:00
+lastmod: 2022-08-15T12:00:35+01:00
+draft: false
+menu:
+  kb:
+    parent: "browse"
+toc: false
+pinned: false
+types: ["article"]
+areas: ["lotus-wallet"]
+---
+
+## Problem:
+
+When installing Lotus for mainnet, lotus-shed is not installed by default. 
```

Comment:
```suggestion
When installing Lotus for mainnet, lotus-shed is not installed by default. 
```
Did you mean lotus-wallet here or do you need shed to install wallet?

Reply:
Meant lotus-wallet ofc. fixed it now